src/panoptes/pocs/filterwheel/filterwheel.py

```diff
@@ -48,6 +52,11 @@ def __init__(self,
         else:
             self._timeout = timeout
         self._serial_number = serial_number
+        if dark_position:
+            self._dark_position = self._parse_position(dark_position)
+        else:
+            self._dark_position = None
+        self._light_position = None
```

Comment:
`_dark_position` is static for the instance but `_light_position` is dynamic.  Maybe `_last_light_position` or something to reflect the difference?

Reply:
Done.